net/dns/resolver/tsdns.go

```diff
@@ -71,7 +72,10 @@ type Config struct {
 	// Queries only match the most specific suffix.
 	// To register a "default route", add an entry for ".".
 	Routes map[dnsname.FQDN][]*dnstype.Resolver
-	// LocalHosts is a map of FQDNs to corresponding IPs.
+	// Hosts maps either domain names or wildcards to IP(s).
+	// If the map key begins with a dot (e.g. ".foo.com.") then it's considered a wildcard,
+	// matching *.foo.com. "foo.com" would not get matched with ".foo.com" since the label "foo"
```

Comment:
this second sentence is still about wild cards but we never gave an example of the normal form of this map key (non-wildcard). Include a little example first of "record.foo.com." (with the trailing dot) so it doesn't look like "foo.com" (which doesn't have a trailing dot in that example) is meant as a map key?